dts/riscv/it8xxx2_evb.dtsi

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * Copyright (c) 2019-2020 Jyunlin Chen <jyunlin.chen@ite.com.tw>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		cpu@0 {
+			clock-frequency = <0>;
+			compatible = "riscv,riscv-ite";
+			device_type = "cpu";
+			reg = <0>;
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+        flash-controller@80000000 {
+            compatible = "ite,it8xxx2-flash-controller";
+            reg = <0x80000000 0x100>;
+            label="fspi";
+            #address-cells = <1>;
+            #size-cells = <1>;
+
+            flash0: flash@80000000 {
+	            compatible = "soc-nv-flash";
+				reg = <0x80000000 DT_SIZE_K(512)>;
+				erase-block-size = <1024>;
+				write-block-size = <1>;
+			};
```

Comment:
Fix indentation for closing brace on this line, line 43, and line 47.

Reply:
This for example hasn't been resolved or addressed.


---
Please go through the list of unresolved comments and mark them as resolved when you fix them.

Reply:
OK, I've closed the resolved comment.
By the way, this comment mention the indentation of  closing brace, but I thought it had already been fixed? (If it hasn't, please let me know. Thanks.)

Reply:
It looks correct now. Thanks!